client/reducers/index.js

```diff
@@ -0,0 +1,49 @@
+import { combineReducers } from 'redux';
+
+import types from "../utils/types";
+
+import {
+  CONVERSION_ERROR,
+  FETCH_CONVERSIONS,
+  SET_CONVERSION_TYPES,
+  SET_CONVERSION_VALUE,
+  UPDATE_CONVERSIONS
+} from '../actions';
+
+function conversions(state = {
+  conversionError: null,
+  conversions: [],
+  types: types.DEFAULT_TYPE,
+  value: ""
+}, action) {
+  switch (action.type) {
+  case CONVERSION_ERROR:
+    return Object.assign({}, state, {
```

Comment:
GLOBAL: Is `Object.assign` polyfillable with es5-shim? Because most of the `Object.*` methods aren't... http://kangax.github.io/compat-table/es5/#es5shim